src/client/Microsoft.Identity.Client/TelemetryCore/TelemetryManager.cs

```diff
@@ -137,7 +138,7 @@ public string FetchAndResetPreviousHttpTelemetryContent()
             {
                 var httpTelemetryContent = new HttpTelemetryContent(_mostRecentStoppedApiEvent);
                 _mostRecentStoppedApiEvent = null;
-                return httpTelemetryContent.GetCsvAsPrevious();
+                return httpTelemetryContent.GetCsvAsPrevious(SuccessfulSilentCallCount);
```

Comment:
The spec mentions that the requests should be deleted if there was an error in receiving by server side; do we handle that scenario?